hw3/01-characters.js

```diff
@@ -1,4 +1,63 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
 
-fetch(url);
+
+const section = document.querySelector('section');
+
+const addCharactersToDOM = (character) =>{
+    let figure = document.createElement('figure');
+    let div = document.createElement('div')
+    let image = document.createElement('img');
+    let figCaption = document.createElement('figcaption');
+    figure.setAttribute('class', 'fig');
+    image.width = "200";
+    image.height = "200";
+    image.alt = `Image ${character.fullName}`;
+    image.setAttribute('src', character.imageUrl);
+    let fullName = document.createElement('p');
+    let title = document.createElement('p');
+
+    fullName.textContent = character.fullName;
+    title.textContent = character.title;
+    figure.append(image);
+    figCaption.append(fullName);
+    figCaption.append(title);
```

Comment:
When using append(), you can append multiple elements at once. It is not necessary, but it would save a line of code.

```suggestion
    figCaption.append(fullName, title);
```